tests/integration/devfile/cmd_devfile_describe_test.go

```diff
@@ -88,6 +95,19 @@ var _ = Describe("odo devfile describe command tests", func() {
 	})
 
 	Context("when running odo describe for machine readable output", func() {
+		When("a component is created with a custom name", func() {
```

Comment:
The `Context` above starts with `when`. Classic example of a test file that's not refactored, and we're adding tests as per our understanding based on #4770.

Reply:
How about we keep it as is for now, and I pick the refactoring in the next PR next sprint along with all the other files?